code/game/objects/items/devices/radio/radio.dm

```diff
@@ -199,6 +199,8 @@
 				return FALSE
 			if(mute.handcuffed)//Would be weird if they couldn't sign but their words still went over the radio
 				return FALSE
+			if(mute.has_status_effect(STATUS_EFFECT_PARALYZED))
```

Comment:
What you want to be checking here is if the mute mob has the `TRAIT_INCAPACITATED` trait, which currently aggregates several conditions, and is set-up by paralyzed as well.

Reply:
That helps a lot, thanks

Reply:
Done